ksqldb-common/src/main/java/io/confluent/ksql/util/VertxSslOptionsFactory.java

```diff
@@ -59,6 +60,13 @@ private static JksOptions buildJksOptions(final Buffer buffer, final String pass
     return new JksOptions().setValue(buffer).setPassword(Strings.nullToEmpty(password));
   }
 
+  private static KeyStoreOptions buildBcfksOptions(final String password) {
+    return new KeyStoreOptions()
+        .setType("BCFKS")
```

Comment:
Just to follow the code in this file, could you add a constant, like the `SSL_STORE_TYPE_JKS` for this too?